### PR TITLE
Get project types

### PR DESCRIPTION
**Pull request by phemmylintry**

### CODE REVIEW

File: openproject/client.py [hunk 1]
```diff
@@ -133,6 +133,9 @@ def update(self, id: int, **kwargs):
     def delete(self, id: int):
         return self.client._send_request("DELETE", f"projects/{id}")
 
+    def types(self, id: int):
```
Comment (parisk) [inline openproject/client.py]: Let's rename this to `list_types` to better represent what it does.

**Review by parisk — CHANGES_REQUESTED** [review]
Just a rename and we should be good to go

File: openproject/client.py (hunk 1 above)
Comment (phemmylintry) [reply openproject/client.py]: okay

**Review by parisk — APPROVED** [review]
LGTM 🐸

**Pull request merged**